README.md

```diff
@@ -1,254 +1,111 @@
 # note-zephyr
 
-A `west` compatible [Zephyr RTOS](https://zephyrproject.org/) module for the [Notecard](https://shop.blues.io/collections/notecard). Include this module in your own project manifest and use the `notecard` module to access the Notecard.
+This is a Zephyr `west` module for the Blues Notecard. It is an abstraction layer for the [note-c](https://github.com/blues/note-c) library, that allows you to easily use a [Notecard](https://blues.com/products/notecard/) in a Zephyr application.
```

Comment:
It's not always practical, but you should attempt to constrain your lines to 80 characters. This makes the README.md legible from a terminal, and not just GitHub.